tests/conftest.py

```
@@ -1399,11 +1455,25 @@ def finalizer():
+                log.warning(f"Failed to strip ownerRefs for {instance.name}: {e}")
+
             try:
                 delete_storageclass_and_deregister(
                     sc_name=instance.name,
                     sc_ocp=instance.ocp,
                 )
+            except CommandFailed as ex:
+                if "NotFound" in str(ex):
+                    log.info(
+                        f"StorageClass {instance.name} not found, "
+                        "skipping deletion"
```

Ruff (0.15.21)

[warning] Logging statement uses f-string

(G004)